torchtext/datasets/ag_news.py

```diff
@@ -29,10 +28,24 @@
 DATASET_NAME = "AG_NEWS"
 
 
-@_add_docstring_header(num_lines=NUM_LINES, num_classes=4)
 @_create_dataset_directory(dataset_name=DATASET_NAME)
 @_wrap_split_argument(("train", "test"))
 def AG_NEWS(root: str, split: Union[Tuple[str], str]):
+    """AG_NEWS Dataset
+
+    For additional details refer to https://paperswithcode.com/dataset/ag-news
+
+    Number of lines per split:
+        train: 120000
+        test: 7600
```

Comment:
It seems we need new line in between them otherwise they will go in same line in build docs. At-least that's what I am seeing on my side. Can you try building docs locally to check this?

Reply:
@parmeet I just updated the splits to be a list. Lmk if this looks okay. I can also create a followup PR to update the docs for the datasets you were working on! 😄

Reply:
Thanks @Nayef211 . List looks much cleaner. Sure please follow-up with #1597 to fix the issue there as well